Poker.BE/Poker.BE.Domain/Core/User.cs

```diff
@@ -8,22 +11,136 @@ namespace Poker.BE.Domain.Core
 {
     public class User : AbstractUser
     {
-        #region Methods
-        public  User (string userName, string password, double sumToDeposit){
+        #region Fields
+        /* singletons */
+        private GameCenter gameCenter = GameCenter.Instance;
+        private ILogger logger = Logger.Instance;
+        /* ---------- */
+
+        #endregion
+
+        #region Properties
+
+        public ICollection<Player> Players { get; set; }
+        #endregion
+
+        #region Constructors
+        public User()
+        {
+            IsConnected = false;
+            Password = "";
```

Comment:
We need to code something else more secure to store password.. I will think about something